packages/frontend/src/view/pages/home/common.ts

```diff
@@ -13,14 +13,14 @@ export const L2_TRANSACTIONS_TABLE_PROPS = {
 }
 
 export const FORCED_TRANSACTION_TABLE_PROPS = {
-  title: 'Latest forced transactions',
+  title: 'Forced transactions',
   entryShortNamePlural: 'transactions',
   entryLongNamePlural: 'forced transactions',
   path: '/forced-transactions',
 }
 
 export const OFFER_TABLE_PROPS = {
-  title: 'Available forced trade offers',
+  title: 'Available offers',
```

Comment:
Naming offers "forced trade offers" was something specifically requested by SW long ago. I'd only make this change if they approved it this way.

Reply:
They've accepted it 👍